specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2019-08-01/notebook.json

```diff
@@ -0,0 +1,446 @@
+          },
+          {
+            "$ref": "../../stable/2019-08-01/cosmos-db.json#/parameters/apiVersionParameter"
+          },
+          {
+            "$ref": "../../stable/2019-08-01/cosmos-db.json#/parameters/accountNameParameter"
+          },
+          {
+            "$ref": "#/parameters/NotebookWorkspaceNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK"
+          },
+          "202": {
+            "description": "Accepted. The start operation will complete asynchronously."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../stable/2019-08-01/cosmos-db.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "NotebookWorkspaceCreateParameters": {
+      "description": "Parameters to create a notebook workspace resource",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../stable/2019-08-01/cosmos-db.json#/definitions/ARMProxyResource"
+        }
+      ]
+    },
+    "NotebookWorkspaceListResult": {
+      "description": "A list of notebook workspace resources",
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "description": "Array of notebook workspace resources",
+          "items": {
+            "$ref": "#/definitions/NotebookWorkspace"
+          }
+        }
+      }
+    },
+    "NotebookWorkspace": {
+      "description": "A notebook workspace resource",
+      "type": "object",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/NotebookWorkspaceProperties",
+          "description": "Resource properties.",
+          "x-ms-client-flatten": true
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../stable/2019-08-01/cosmos-db.json#/definitions/ARMProxyResource"
+        }
+      ]
+    },
+    "NotebookWorkspaceProperties": {
+      "description": "Properties of a notebook workspace resource.",
+      "type": "object",
+      "properties": {
+        "notebookServerEndpoint": {
+          "description": "Specifies the endpoint of Notebook server.",
+          "type": "string",
+          "readOnly": true
+        },
+        "status": {
+          "description": "Status of the notebook workspace. Possible values are: Creating, Online, Deleting, Failed, Updating.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "NotebookWorkspaceConnectionInfoResult": {
+      "description": "The connection info for the given notebook workspace",
+      "properties": {
+        "authToken": {
+          "type": "string",
+          "description": "Specifies auth token used for connecting to Notebook server (uses token-based auth)."
+        },
+        "notebookServerEndpoint": {
```

Comment:
>"notebookServerEndpoint" [](start = 8, length = 24)

is there a reason to replicate this on the resource as well as this connectionInfo result?

Reply:
We felt the notebook server endpoint property is relevant in both context so it is added as a convenience for the user. If you strongly feel it is redundant, we can eliminate one of them.